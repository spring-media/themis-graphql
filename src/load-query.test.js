const { loadQuery } = require('./load-query');
const path = require('path');

describe('load-query', () => {
  it('handles a file with recursive #import statements', async () => {
    const result = await loadQuery(path.resolve(__dirname, '../test/data/queries/start.gql'));
    expect(result).toMatchSnapshot();
  });
});