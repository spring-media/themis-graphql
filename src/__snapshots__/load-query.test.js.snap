// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`load-query handles a file with recursive #import statements 1`] = `
"CCC
CCC
CCC
DDD
DDD
DDD
BBB
BBB
BBB
AAA
AAA
AAA"
`;
